sonar-jira/src/main/java/sonar/api/HelloWorld.java

```diff
@@ -0,0 +1,11 @@
+package sonar.api;
+
+/**
+ * Created by awong on 12/07/2016.
+ */
+public class HelloWorld {
+
+    public HelloWorld() {
+        System.out.println("hello world.");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS106)